lib/tui/widget/common.ml

```diff
@@ -4,21 +4,24 @@ let fmt_error (error : Gh.Client.error) =
   | No_github_token ->
       [
         str
-          "\u{26A0} GITHUB_TOKEN not found. Make sure it's configured in your \
-           environment.";
+          (Pretty.Icon.warning
+         ^ " GITHUB_TOKEN not found. Make sure it's configured in your \
+            environment.");
         str "";
         str "If you don't have a token, visit the following page to create one:";
         str "  • https://github.com/settings/tokens";
       ]
   | Bad_credentials { msg; doc_url; code } ->
       [
-        str (Format.sprintf "\u{26A0} [%d] %s" code msg);
+        str (Pretty.Icon.warning ^ Format.sprintf " [%d] %s" code msg);
```

Comment:
Since we're already using `sprintf` here, let's leverage it for icons as well:

```suggestion
        str (Format.sprintf "%s [%d] %s" Pretty.Icon.warning code msg);
```